### PR TITLE
Update Title on page change

### LINKED ISSUE #80: Change UI title according to page

### PR DESCRIPTION
**Pull request by sogggy**

### CODE REVIEW

**Comment by sogggy** [comment]
Close #80

File: src/main/java/seedu/address/logic/LogicManager.java [hunk 1]
```diff
@@ -90,8 +91,23 @@ public GuiSettings getGuiSettings() {
...
+        return this.model.getPageTitle();
+    }
+
     @Override
     public void setGuiSettings(GuiSettings guiSettings) {
         model.setGuiSettings(guiSettings);
     }
+
+    @Override
+    public boolean getIsExpenditure() {
```
Comment (wenhaogoh) [inline src/main/java/seedu/address/logic/LogicManager.java]: Perhaps the naming should be "isExpenditure"?

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -128,16 +129,26 @@ void fillInnerParts() {
...
         //statusbarPlaceholder.getChildren().add(statusBarFooter.getRoot());
 
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
     }
 
+    void bindTitleToState() {
+        title.getTitle().textProperty().bind(Bindings.createStringBinding(() -> {
+            if (logic.getIsExpenditure()) {
+                return logic.getPageTitle();
```
Comment (wenhaogoh) [inline src/main/java/seedu/address/ui/MainWindow.java]: Is this condition redundant?

**Review by wenhaogoh — CHANGES_REQUESTED** [review]

**Review by wenhaogoh — APPROVED** [review]
LGTM

**Review by davidliew9 — APPROVED** [review]
LGTM

**Pull request merged**